### DISCUSSION TITLE
Add github discussions

### DISCUSSION THREAD

**Discussion by pagrubel** [lanl/BEE#369] [General]
Al pointed github discussions out https://docs.github.com/en/discussions I'm making it an issue so we can remember to add it when we go public. Also any discussion about it is welcome.

**Comment by mcpherson** [comment]
Might even be useful for `BEE_Private`. We often use issues for things that don't relate to the code (design discussions, etc.). This very issue is a case in point 😉.

**Comment by pagrubel** [comment]
Okay, I converted this to a discussion to show that discussions are now a thing in BEE_Private.
